activerecord/lib/active_record/connection_adapters/abstract/schema_definitions.rb

```diff
@@ -303,7 +303,7 @@ def [](name)
       #   end
       def column(name, type, options = {})
         name = name.to_s
-        type = type.to_sym
+        type = type ? type.to_sym : :__placeholder
```

Comment:
@rafaelfranca It calls `type.to_sym` one more time before finishing so I would have to go in and also handle the nil case there to fix.  Would that be preferable to this solution?  Can fix it quickly if so.